hello-strongbox-gradle/build.gradle

```diff
@@ -0,0 +1,65 @@
+apply plugin: 'java'
+apply plugin: 'maven'
+
+compileJava {
+    targetCompatibility = "1.8"
+    sourceCompatibility = "1.8"
+    options.encoding = 'UTF-8'
+}
+
+group = 'org.carlspring.strongbox.examples'
+version = '1.0'
+
+description = "Example project that shows how to use gradle to build and deploy artifacts to Strongbox"
+
+configurations {
+    deployerJars
+}
+
+repositories {
+    maven { url "http://dev.carlspring.org/nexus/content/groups/carlspring" }
```

Comment:
@AlexOreshkevich,
Right, so I suppose it uses this to check if the `maven-metadata.xml` and artifact have been deployed...?